js/main.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+function getRandomInteger(min, max) {
+  return Math.floor(Math.random() * (max - min)) + min;
+}
+
+function getRandomComment() {
+  var COMMENTS = [
+    'Всё отлично!',
+    'В целом всё неплохо. Но не всё.',
+    'Когда вы делаете фотографию, хорошо бы убирать палец из кадра. В конце концов это просто непрофессионально.',
+    'Моя бабушка случайно чихнула с фотоаппаратом в руках и у неё получилась фотография лучше.',
+    'Я поскользнулся на банановой кожуре и уронил фотоаппарат на кота и у меня получилась фотография лучше.',
+    'Лица у людей на фотке перекошены, как будто их избивают. Как можно было поймать такой неудачный момент?!'
+  ];
+
+  return COMMENTS[getRandomInteger(0, COMMENTS.length - 1)] + ' ' + COMMENTS[getRandomInteger(0, COMMENTS.length - 1)];
+}
+
+function getRandomName() {
+  var NAMES = [
+    'Артем',
+    'Наташка'
+  ];
+
+  return NAMES[getRandomInteger(0, NAMES.length - 1)];
+}
+
+function getRandomComments(quantity) {
+  var comments = [];
+  var comment = {};
+
+  for (var i = 1; i <= quantity; i++) {
+    comment = {
+      avatar: 'img/avatar-' + getRandomInteger(1, 6) + '.svg',
+      message: getRandomComment(),
+      name: getRandomName()
+    };
+
+    comments.push(comment);
+  }
+
+  return comments;
+}
+
+function getRandomDataList(length) {
+  var dataList = [];
+  var el = {};
+
+  for (var i = 1; i <= length; i++) {
+    el = {
+      url: 'photos/' + i + '.jpg',
+      likes: getRandomInteger(15, 200),
+      comments: getRandomComments(getRandomInteger(1, 5))
+    };
+
+    dataList.push(el);
+  }
+
+  return dataList;
+}
+
+
+function renderPictures(dataList) {
+  var pictureTemplate = document.getElementById('picture').content.children[0].cloneNode(true);
+  var picturesListFragment = document.createDocumentFragment();
+  var picturesListElement = document.getElementsByClassName('pictures')[0];
+
+  for (var i = 1; i <= dataList.length; i++) {
+    var picture = pictureTemplate.cloneNode(true);
+
+    picture.getElementsByClassName('picture__img')[0].setAttribute('src', dataList[i - 1].url);
```

Comment:
```suggestion
    picture.querySelector('.picture__img').src = dataList[i - 1].url
```

Вместе `getElementsByClassName` используй `querySelector`

Reply:
и так далее по коду